cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java

```diff
@@ -27,6 +27,13 @@
 
     public static void main(String[] args) throws Exception
     {
+        
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
```

Comment:
```suggestion
        if (Float.parseFloat(System.getProperty("java.class.version")) < 52F) 
```

Please compare floats to floats